content/notebooks/hydrology/03_Watershed_properties.ipynb

```diff
@@ -0,0 +1,758 @@
+{
```

Comment:
<p><code>band_data</code> var missing here as well for the <code>dem</code> Dataset  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-landing/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>